mmf/configs/zoo/models.yaml

```diff
@@ -1,12 +1,105 @@
+          file_name: visual_bert.pretrained.vqa2_fifty_pc.tar.gz
+          hashcode: 9fdaa945cb727f422262e770c72c3dc897d5ec172a5049b559ae2cbd7b997bec
+      ten_pc:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.vqa2_ten_pc.tar.gz
+          file_name: visual_bert.pretrained.vqa2_ten_pc.tar.gz
+          hashcode: f6a20a4946d94908d456c3d67bc18b8651a25767553ee1ef3a8c8875b797dd07
+      full_train_val:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.vqa2_train_val.tar.gz
+          file_name: visual_bert.pretrained.vqa2_train_val.tar.gz
+          hashcode: 017b506db1ba2b3ae27e73f00ddcc2c52a89e59922238c5b5ae053b48c8cb590
+    cc:
+      full:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_full.tar.gz
+          file_name: visual_bert.pretrained.cc_full.tar.gz
+          hashcode: 4351f9618a71b4fa9f52958d6742976ee0f31cae41714f1021e5cbca8aa1d31f
+      half:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_half.tar.gz
+          file_name: visual_bert.pretrained.cc_half.tar.gz
+          hashcode: 96e15d6630aea7f5a6e96a2ce39dba49c23ef0d60ce5046bb684d0495fa8c95f
+      small:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_small.tar.gz
+          file_name: visual_bert.pretrained.cc_small.tar.gz
+          hashcode: 5f3d5907a828d8381191ee1d5fc28100bcc8410fd01ff7675a4d4434ae796a42
+      small_fifty_pc:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_small_fifty_pc.tar.gz
+          file_name: visual_bert.pretrained.cc_small_fifty_pc.tar.gz
+          hashcode: c510b33ad9c1de9da313d0f5624e24ebdc58de34bd91fd73bbae30ae1d0bb4be
+      small_ten_pc:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_small_ten_pc.tar.gz
+          file_name: visual_bert.pretrained.cc_small_ten_pc.tar.gz
+          hashcode: 5c701e7f041e371a6241cfe9fafe66c2b4da0323a0d9a349cf50d42b59bf0ed7
+      full_coco_generated:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_full_coco_generated.tar.gz
+          file_name: visual_bert.pretrained.cc_full_coco_generated.tar.gz
+          hashcode: 27acc32ab55a9f771d354fec8c2c76fed3464fa0b569dd465b0552c0a79afca5
+      half_coco_generated:
+        version: 1.0_2020_05_19
+        resources:
+        - url: mmf://models/visual_bert/visual_bert.pretrained.cc_half_coco_generated.tar.gz
+          file_name: visual_bert.pretrained.cc_half_coco_generated.tar.gz
+          hashcode: a35cdfab35bcb23956bff29f38253aeb4adbec5518ea0a023232f0107f827261
+      small_coco_generated:
```

Comment:
Can we add a comment here on what exactly is small and half? and link to the paper?